model/build.gradle

```diff
@@ -14,6 +14,11 @@ dependencies {
     compile libraries.jackson.core
 
     compile libraries.swagger.annotations
+
+    // Primitive Collection Library for speed and lower-memory
+    // footprint
+    // UPGRADE to latest version when we can use Java 8
+    compile 'it.unimi.dsi:fastutil:7.2.1'
```

Comment:
We are now building with Java 8.

Reply:
This was a copy-paste from the other module. Should we upgrade both to use the latest?

Reply:
Yes that would probably be a good idea. You can define the dependency in libraries.gradle to keep the dependencies in sync.